src/20_Components/amp-live-list.html

```diff
@@ -0,0 +1,147 @@
+<!---{
+    "experiment": true,
+    "component": "amp-live-list",
+    "preview": true
+  }--->
+<!--
+  #### Introduction
+
+  The [amp-live-list](https://github.com/ampproject/amphtml/blob/f2aed679a4c0611ea077985cbe0d1926c7d0a6f5/extensions/amp-live-list/amp-live-list.md) component add support for publishing content that updates live in the AMP page as new content is available in the source document.
+
+  This example shows an implementation with a server rendering a series of 10 elements, `amp-live-list` will poll the page every 15 seconds. Every time `amp-live-list` polls the page, the server adds a new item to the page. Refer to [server.go](https://github.com/ampproject/amp-by-example/blob/master/server.go) for implementation details.
+
+  When the user lands to this page the first time, the server returns one item and creates a cookie containing the number of times the user has accessed the page (it will be 1 on creation). For every other access, this number is incremented by one and a new item is added to the page server side. 
```

Comment:
Just say: "The sample live blog stops after ten articles. If you want to start over, delete cookies from ampbyexample.com or open this page again in an incognito/private tab."